AzureTable/Microsoft.DataTransfer.AzureTable/Microsoft.DataTransfer.AzureTable.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.DataTransfer.AzureTable</RootNamespace>
     <AssemblyName>Microsoft.DataTransfer.AzureTable</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```

Comment:
We actually have `Solution Items\CommonProperties.proj`, which is imported to all of the projects (except tests). I'm wondering, if it makes sense to add it there instead, so that it will be a simpler change next time?

Reply:
that's a great idea. I'll make that change, and push another commit

Reply:
update: I tried the change, and it does not seem to work with Visual Studio. Upon doing so, the project version always defaults to 4.0. Hence, keeping it as is for now.